snapshot/android/utils.js

```diff
@@ -12,49 +13,34 @@ const createDirectory = dir => mkdir('-p', dir);
 
 const downloadFile = (url, destinationFilePath) =>
     new Promise((resolve, reject) => {
-        const request = httpsGet(url, response => {
-            switch (response.statusCode) {
-                case 200:
-                    const file = createWriteStream(destinationFilePath, {autoClose: true});
-                    file.on('error', function (error) {
-                        return reject(error);
-                    });
-                    file.on("finish", function() {
-                        chmodSync(destinationFilePath, 0755);
-                        return resolve(destinationFilePath);
-                    });
-                    response.pipe(file);
-                    break;
-                case 301:
-                case 302:
-                case 303:
-                    const redirectUrl = response.headers.location;
-                    return this.downloadExecFile(redirectUrl, destinationFilePath);
-                default:
-                    return reject(new Error("Unable to download file at " + url + ". Status code: " + response.statusCode));
-            }
-        });
-
-        request.end();
-
-        request.on('error', function(err) {
-            return reject(err);
-        });
+        get(url)
```

Comment:
any tests for this functionality are preferable

Reply:
I left it without tests, because I thought that the tests for the `get` method inside the `request` package are enough. I agree that it would be good to test our specific functionality. Can you suggest some scenarios that I can add?